docs/en/overview/JobService.md

```diff
@@ -59,6 +59,13 @@ to a specified number of other workers.
 The job service also supports data transformation. More details can be found at 
 [Data Transformations]({{ '/en/core-services/Transformation.html' | relativize_url }})
 
+### Async Submission
+
+The job service now supports async submission mode for distributedLoad and distributedCp commands.
```

Comment:
- remove `now`. it doesn't make sense several versions from now to continue to read "the job service now supports xyz"
- is async submission only going to be for distributedLoad and distributedCp commands? what is the probability that new commands can be async and multiply that by the probability this statement in the doc will be reviewed and updated. for an overview doc, it is better to explain in more general terms to follow the tone of the doc. i would not explain exactly which commands. if it is not a majority of operational commands, i would word it as certain key commands.

Reply:
Async submission is only for distributedLoad and distributedCp now. We are not sure whether other commands can be async. Maybe I will just remove this from this overview doc currently.